build.assets/makefiles/credential-provider/credential-provider.mk

```diff
@@ -0,0 +1,15 @@
+.PHONY: all
+
+BIN := $(ASSETDIR)/ecr-credential-provider
+
+all: $(BIN)
+	@echo "\n---> Installing ecr-credential-provider:\n"
+	mkdir -p /opt/ecr-credential-provider/bin /etc/ecr-credential-provider
```

Comment:
```suggestion
	mkdir -p $(ROOTFS)/opt/ecr-credential-provider/bin $(ROOTFS)/etc/ecr-credential-provider
```

Reply:
Thanks this fixed a few errors I was getting.

Reply:
Updated https://github.com/gravitational/planet/pull/883/commits/a5c741a6aae818d3af2cc4f0d01e753c387d6aec